conf/docker.properties.template

```diff
@@ -0,0 +1,3 @@
+spark.executor.docker.image: amplab/spark-1.1.0
```

Comment:
I think once I fix up the code to show how to build an image from the result of `make-distribution.sh` the example image name can reflect the name of the built image. That current name is really just a placeholder, I don't think such an image actually exists!